lisa/messages.py

```diff
@@ -137,3 +137,60 @@ class NetworkPPSPerformanceMessage(PerfMessage):
     rx_tx_pps_minimum: Decimal = Decimal(0)
     rx_tx_pps_average: Decimal = Decimal(0)
     rx_tx_pps_maximum: Decimal = Decimal(0)
+
+
+@dataclass
+class NetworkNtttcpTCPPerformanceMessage(PerfMessage):
+    test_case_name: str = ""
+    test_date: datetime = datetime.utcnow()
+    platform: str = ""
+    location: str = ""
+    host_version: str = ""
+    guest_os_type: str = "Linux"
+    distro_version: str = ""
+    vmsize: str = ""
+    kernel_version: str = ""
+    lis_version: str = ""
+    ip_version: str = "IPv4"
+    protocol_type: str = "TCP"
+    data_path: str = ""
+    connections_num: int = 0
+    throughput_in_gbps: Decimal = Decimal(0)
+    latency_us: Decimal = Decimal(0)
+    buffer_size: Decimal = Decimal(0)
+    tx_packets: Decimal = Decimal(0)
+    rx_packets: Decimal = Decimal(0)
+    pkts_interrupts: Decimal = Decimal(0)
+    number_of_receivers: int = 1
+    number_of_senders: int = 1
+    sender_cycles_per_byte: Decimal = Decimal(0)
+    connections_created_time: int = 0
+    retrans_segments: int = 0
+    receiver_cycles_rer_byte: Decimal = Decimal(0)
+
+
+@dataclass
+class NetworkNtttcpUDPPerformanceMessage(PerfMessage):
+    test_case_name: str = ""
+    test_date: datetime = datetime.utcnow()
+    platform: str = ""
+    location: str = ""
+    host_version: str = ""
```

Comment:
The host_version, location and some other fields can be defined in PerfMessage.

Reply:
move common ones into PerfMessage.